bin/add-prettier-scripts.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const packageJson = require('../package.json');
+
+const { name } = packageJson;
+
+/**
+ * Adds Prettier scripts to the package.json file
+ */
+const addPrettierScripts = () => {
+  console.info('Adding Prettier scripts to package.json...');
+
+  const packageJsonPath = path.join(process.cwd(), 'package.json');
+  if (!fs.existsSync(packageJsonPath)) {
+    console.error('Error: package.json not found in the current directory');
+    process.exit(1);
+  }
+
+  try {
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+
+    packageJson.scripts = {
+      ...packageJson.scripts,
+      'prettier:check': 'prettier --check "**/*.{ts,tsx,js,jsx}" "**/*.json" "**/*.md"',
+      'prettier:write': 'prettier --write "**/*.{ts,tsx,js,jsx}" "**/*.json" "**/*.md"',
```

Comment:
Since we've got a `.prettierignore` file they recommend to use `prettier --write .` etc for simplicity.

Prettier formats .css, .scss and [some others](https://prettier.io/docs/) that we're losing by not opting into atm - imo opting out is simpler.

```suggestion
      'prettier:check': 'prettier --check .',
      'prettier:write': 'prettier --write .',
```